Lib/multiprocessing/pool.py

```diff
@@ -233,6 +265,8 @@ def __del__(self, _warn=warnings.warn, RUN=RUN):
         if self._state == RUN:
             _warn(f"unclosed running multiprocessing pool {self!r}",
                   ResourceWarning, source=self)
+            if getattr(self, '_change_notifier') is not None:
```

Comment:
Why not just self._change_notifier?

getattr(obj, attr) raises AttributeError if the attribute doesn't exist. Maybe you want to write getattr(self, '_change_notifier', None)?